apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -170,8 +169,8 @@ cleanup_genesis(Val) ->
     Val.
 
 add_missing_to_genesis_block(#{<<"height">> := 0} = Block) ->
-    GB = aec_blocks:serialize_to_map(
-           aec_block_genesis:genesis_block_as_deserialized_from_network()),
+    {ok, GenesisBlock} = aec_chain:genesis_block(),
```

Comment:
It looks like this change is now moving across process heaps a potentially large term - it being the in-memory trees.

Reply:
And actually this line - by inspection of `empty_fields_in_genesis()` - only needs the genesis header plus the transactions, and the transactions in genesis can be assumed to be empty in this point.

No need to change so far - I need to check aec_chain implementation.

Reply:
Opened https://www.pivotaltracker.com/story/show/153140910